packages/apputils/style/mainareawidget.css

```diff
@@ -4,6 +4,6 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 
-.jp-MainAreaWidget:focus {
+.jp-MainAreaWidget > *:focus {
```

Comment:
If the intention is *any immediate descendant of `.jp-MainAreaWidget` that has focus*, then the `*` is unnecessary and `.jp-MainAreaWidget > :focus` is sufficient.